src/etl/usecase/Save_Data.py

```
@@ -0,0 +1,29 @@
+
+    def __init__(self, grabber: TelegramGrabber, repository: IRepository, yd_parser: YandexParser, html_grabber: HTMLGrabber):
+        self.grabber = grabber
+        self.repository = repository
+        self.yd_parser = yd_parser
+        self.html_grab = html_grabber
+
+    
+    async def execute(self, source_type: str, source: str):
+
+        # 1. Получаем сообщения (уже MessageMetadata)/Яндекс Диск
+        # В TelegramGrabber и YandexGrabber уже выполняется и парсинг и перевод в текста в сущность.
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `В` (CYRILLIC CAPITAL LETTER VE). Did you mean `B` (LATIN CAPITAL LETTER B)?

(RUF003)